libs/common/src/helpers/is-gef-facility-type.ts

```diff
@@ -0,0 +1,9 @@
+import { FACILITY_TYPE } from '../constants';
+import { FacilityType } from '../types';
+
+/**
+ * A facility belongs to a GEF deal if and only if the facility type is 'CASH' or 'CONTINGENT'.
+ *
+ * If a facility has the type 'LOAN' or 'BOND' then it belongs to a BSS/EWCS deal.
+ */
```

Comment:
can the params and return type also be included please?

eg 
`
 * @param {FacilityType} type: ...
 * @returns {Boolean} ....
`

Reply:
Refactored, this file no longer features in this PR